modules/nw-ovn-k-day-2-masq-subnet.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-ovn-k-day-2-masq-subnet_{context}"]
-= Configuring the OVN-Kubernetes masquerade subnet as a Day 2 operation
+= Configuring the OVN-Kubernetes masquerade subnet as a post-installation operation
 
-You can change the masquerade subnet used by OVN-Kubernetes as a Day 2 operation in order to avoid conflicts with any existing subnets that are already in use in your environment.
+You can change the masquerade subnet used by OVN-Kubernetes as a post-installation operation in order to avoid conflicts with any existing subnets that are already in use in your environment.
```

Comment:
```suggestion
You can change the masquerade subnet used by OVN-Kubernetes as a post-installation operation to avoid conflicts with any existing subnets that are already in use in your environment.
```

Reply:
:heavy_check_mark: